File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/pythonorganization/donot/openme12/graph/SubGraph_2/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from pythonorganization.donot.openme12.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(col("`c- short`"), col("p_timestamp"))

File: test/sony/scala/livy/pipelines/minimal/code/src/main/scala/io/prophecy/pipelines/minimal/graph/Reformat_1.scala
package io.prophecy.pipelines.minimal.graph

import io.prophecy.libs._
import io.prophecy.pipelines.minimal.config.ConfigStore._
import io.prophecy.pipelines.minimal.config.Context
import io.prophecy.pipelines.minimal.udfs.UDFs._
import io.prophecy.pipelines.minimal.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._
object Reformat_1 { def apply(context: Context, in: DataFrame): DataFrame = in }
